com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -499,23 +666,39 @@ void DrawStackSettings()
                 m_LayerList.DoLayoutList();
                 m_SerializedCamera.serializedObject.ApplyModifiedProperties();
 
-                if (errorCameras.Any())
+                EditorGUI.indentLevel--;
```

Comment:
I think we shouldn't indent up the helpboxes, I think you should indent the list. @TheoWong-pixel 

![image](https://user-images.githubusercontent.com/76204843/110335223-e8a0e000-8023-11eb-82f8-2cb6d79c7c76.png)

You can use this to align the list properly:


```
var rect = GUILayoutUtility.GetRect(1, m_LayerList.GetHeight(), GUILayout.ExpandWidth(true));
m_LayerList.DoList(EditorGUI.IndentedRect(rect));
```